test_unstructured_ingest/expected-structured-output/confluence-diff/MFS/1540126.json

```diff
@@ -84,7 +84,7 @@
         "eng"
       ],
       "page_number": 1,
-      "text_as_html": "<table><br><tbody><br><tr><td>Must have:   </td></tr><br><tr><td>Nice to have:</td></tr><br><tr><td>Not in scope:</td></tr><br></tbody><br></table>"
+      "text_as_html": "<table><tr><td>Must have:</td></tr><tr><td>Nice to have:</td></tr><tr><td>Not in scope:</td></tr></table>"
```

Comment:
This one demonstrates the new `.strip()` behavior of cell contents. Leading and trailing space in an HTML cell is not meaningful and not rendered so another opportunity to compact the already longer-than-.text `.text_as_html` value. This length mismatch is especially important when it comes to chunking.